papaya-core/src/main/java/pl/edu/agh/papaya/service/jira/JiraAuthorisationService.java

```diff
@@ -0,0 +1,82 @@
+package pl.edu.agh.papaya.service.jira;
+
+import com.google.api.client.auth.oauth.OAuthParameters;
+import com.google.api.client.http.GenericUrl;
+import com.google.api.client.http.HttpRequest;
+import com.google.api.client.http.HttpRequestFactory;
+import com.google.api.client.http.HttpResponse;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.security.NoSuchAlgorithmException;
+import java.security.spec.InvalidKeySpecException;
+import java.util.Optional;
+import java.util.Scanner;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.json.JSONObject;
+import org.springframework.stereotype.Service;
+import pl.edu.agh.papaya.service.project.ProjectService;
+
+@Service
+@Getter
+@Setter
+@NoArgsConstructor
+@SuppressWarnings("SummaryJavadoc")
+public class JiraAuthorisationService {
+
+    //todo: Passing via @Value
+    private static final String PRIVATE_KEY = "";
+    private static final String CONSUMER_KEY = "";
+
+    private ProjectService projectService;
```

Comment:
To jest używane gdzieś? Nie jest `@Autowired` ani nie ma `@RequiredArgsConstructor`

Reply:
Usunięto `projectService`